.github/workflows/build_hub.yml

```diff
@@ -51,16 +48,6 @@ jobs:
         run: |
           echo >> $GITHUB_ENV DB_USER=${POSTGRES_USER}
           echo >> $GITHUB_ENV DB_PASSWORD=${POSTGRES_PASSWORD}
-          if [ "$GITHUB_BASE_REF" == "production" ]
-          then
-              echo "Building for the production environment."
-              echo >> $GITHUB_ENV ACR_REPO=pdhprodcontainerregistry.azurecr.io
-              echo >> $GITHUB_ENV PREFIX=pdhprod
-          else
-              echo "Building for the test environment."
-              echo >> $GITHUB_ENV ACR_REPO=pdhstagingcontainerregistry.azurecr.io
-              echo >> $GITHUB_ENV PREFIX=pdhstaging
-          fi
```

Comment:
There are old unused environment variables. Before we had our Azure service principal defined here as well because the CI job used to also do CD instead of being a separate workflow

Reply:
This was wrong. They're still needed, but they're not secrets. Opted to keep them there for now and we can address in a future PR.